google-cloud-logging/src/main/java/com/google/cloud/logging/spi/v2/LoggingRpc.java

```diff
@@ -93,6 +95,18 @@
    */
   ApiFuture<Empty> delete(DeleteSinkRequest request);
 
+  /**
+   * Sends a request to list the log names in a project. This method returns a {@code ApiFuture}
+   * object to consume the result. {@link ApiFuture#get()} returns a response object containing the
+   * listing result.
+   *
+   * @param request the request object containing all of the parameters for the API call
+   */
+  default ApiFuture<ListLogsResponse> list(ListLogsRequest request) {
```

Comment:
This should probably be renamed to `listLogs` or `listLogNames`. There's already a `list()` method which is poorly named for listing entries. This now becomes an overload of the same method name with different functionality.

Reply:
I've tried to followed the existing pattern. However, I will rename the method to be `listLogs`.